microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api.md

```diff
@@ -35,17 +35,27 @@ The Payment Request API⁠⁠ handles the actual payment transaction when a purc
 See [Payment Request API](https://developer.mozilla.org/docs/Web/API/Payment_Request_API) at MDN.
 
 
+<!-- ====================================================================== -->
+## Origin trial and flag
+
+There's an origin trial and Edge flag for this feature.  As of December 1, 2024, this origin trial is likely to end soon, graduating the feature to be incorporated in Microsoft Edge.  See [Experimental features and origin trials for PWAs](https://learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/origin-trials).
+
+The Edge Experiments flag in `edge://flags/`<!-- todo: which flag? --> will be enabled by default, when this feature ships in Edge Stable.
```

Comment:
There are three feature flags: `msWebAppDigitalGoods`, `AppStoreBilling` and `DigitalGoodsApi`.
These feature flags are intentionally excluded from the `edge://flags` page, because we do not want users to enable or disable them manually.

Reply:
Strategy: Remove this section "Origin trial and flag", and then merge this PR now.

In 3 weeks, create a PR that revises section:
https://learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api#checking-whether-the-digital-goods-api-is-available
Rewrite "you've correctly enabled the API on your website" like:
to detect whether the API is available to your app, for your app to use, ...

Reply:
snapshot of result:
```
## Origin trial and flag

There's an origin trial and Microsoft Edge flag for this feature.  See [Digital Goods API]
(https://developer.microsoft.com/microsoft-edge/origin-trials/trials/4b4a9ead-d912-4349-87b3-25e5e50b4f13) 
at _Microsoft Edge Developer_ > _Origin Trials_.

The Edge Experiments flag in `edge://flags/`<!-- todo: which flag? --> will be enabled 
by default, when this feature ships in Edge Stable.
```
https://developer.microsoft.com/microsoft-edge/origin-trials/trials/4b4a9ead-d912-4349-87b3-25e5e50b4f13

Reply:
probably Outdated; will Close and re-enter Comment as needed, at each "todo"